anndata/tests/helpers.py

```diff
@@ -476,3 +493,18 @@ def fmt_name(x):
             exact,
             elem_name=fmt_name(attr),
         )
+
+
+def darr_from_arr(arr):
+    import dask.array as da
+
+    return da.from_array(arr, chunks="auto")
+
+
+def darr_from_arr_dense(arr):
+    import dask.array as da
+    import scipy
+
+    if scipy.sparse.issparse(arr):
+        return da.from_array(asarray(arr.toarray()), chunks="auto")
```

Comment:
Ah, I shouldn't have committed this version. So we were trying to see where it fails on a test with Rahul and I wanted to see if it worked or not. 

The new commit has what I wanted to do. Why do I think it is useful? Because it helped me and Rahul catch some bugs.